### ISSUE TITLE
debian repository

### ISSUE THREAD

**Issue by 532910** [stoically/radical-native#22]
Please add deb to official debian repository or packages.riot.im

**Comment by 532910** [comment]
I have my own apt repository where I can put deb files. But not sure this is a good production solution.

**Comment by stoically** [comment]
Yeah, thanks for the offer, but I'd prefer something generic. I've read that it might be possible to deploy a repo on GitHub Pages, not sure if that's still a thing.
